Physics/Intersection.h

```diff
@@ -31,7 +31,8 @@ class Intersection {
 	static float RayWithPaddedAabb(const glm::vec3& rayStart, const glm::vec3& rayVec, const BoundingBox& aabb, float padding);
 	static float RayWithPaddedTriangle(const glm::vec3& rayStart, const glm::vec3& rayDir, const glm::vec3& v1, const glm::vec3& v2, const glm::vec3& v3, float padding);
 
-	static bool FrustumWithAabb(const Frustum& frustum, const BoundingBox& aabb);
+	static bool FrustumPlaneWithAabb(BoundingBox& aabb, const glm::vec3& normal, const float distance);
+	static bool FrustumWithAabb(const Frustum& frustum, BoundingBox& aabb);
```

Comment:
Frustum is only forward declared in this file, yet used here as a reference instead of a pointer.
A different include order can cause this to give errors.
Move Frustum into its own file and include it